profiling/src/lib.rs

```diff
@@ -475,13 +479,19 @@ extern "C" fn rinit(_type: c_int, _module_number: c_int) -> ZendResult {
         output_pprof,
     ) = unsafe {
         let profiling_enabled = config::profiling_enabled();
+        let profiling_experimental_features_enabled =
+            config::profiling_experimental_features_enabled();
         (
             profiling_enabled,
+            profiling_enabled && profiling_experimental_features_enabled,
             profiling_enabled && config::profiling_endpoint_collection_enabled(),
-            profiling_enabled && config::profiling_experimental_cpu_time_enabled(),
+            profiling_experimental_features_enabled
+                || profiling_enabled && config::profiling_experimental_cpu_time_enabled(),
             profiling_enabled && config::profiling_allocation_enabled(),
-            profiling_enabled && config::profiling_experimental_timeline_enabled(),
-            profiling_enabled && config::profiling_experimental_exception_enabled(),
+            profiling_experimental_features_enabled
+                || profiling_enabled && config::profiling_experimental_timeline_enabled(),
+            profiling_experimental_features_enabled
+                || profiling_enabled && config::profiling_experimental_exception_enabled(),
             config::profiling_experimental_exception_sampling_distance(),
```

Comment:
Was this line missed?

Reply:
Yes, but on purpose 😉 
This does not enable or disable any feature, but get the sampling distance.